src/main/java/careconnect/ui/PersonListPanel.java

```diff
@@ -24,20 +24,25 @@ public class PersonListPanel extends UiPart<Region> {
     /**
      * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
      */
-    public PersonListPanel(ObservableList<Person> personList, Consumer<Integer> showSelectedPerson) {
+    public PersonListPanel(ObservableList<Person> personList, Consumer<Integer> setAndShowSelectedPerson) {
         super(FXML);
         personListView.setItems(personList);
```

Comment:
Do we need to add `this.personListView...` ? Cuz there was `this` in the other uses of `personListView